echo.js

```diff
@@ -1,7 +1,11 @@
 function echo(){
-  // ...
+  for (var i = 0; i < arguments.length; i++){
+  {console.log(arguments[i]); }
+}
```

Comment:
Careful with your indentation! Also, the inner set of curly braces aren't needed.